docs: describe how to use podman with kind (#202)
kubernetes-sigs/cloud-provider-kind: README.md

policy-local-59854877c9-xwtfk   1/1     Running   0          2m38s
```

> ### Podman Support

Reviewer: I think this will be better if we do a section for the `--enable-lb-port-mapping` and explain that applies to Podman when NOT running as root, since it applied to running cloud-provider-kind as a pod in Windows or mac too

```suggestion
### Enabling Load Balancer Port Mapping
```

> Podman does not allow binding to privileged ports (e.g., 1-1024) unless it's

Reviewer: ```suggestion
When running cloud-provider-kind in a container on Windows or macOS, accessing LoadBalancer services can be challenging. Similar problem happens when not running Podman as root , since does not allow binding to privileged ports (e.g., 1-1024). The -enable-lb-port-mapping flag provides a solution by enabling the necessary port mapping, allowing host access to these services.
```

running as root, which is not recommended.

To connect to your service when using Podman, run `cloud-provider-kind` with the

> `-enable-lb-port-mapping` option. This configures the Envoy container with an

Reviewer 2: ```suggestion
`--enable-lb-port-mapping` option. This configures the Envoy container with an
```

Author: The `--help` option lists the options with a single dash. That's the only reason I used it that way. Happy to change though.

```
> cloud-provider-kind --help
Usage: cloud-provider-kind [options]

  -enable-lb-port-mapping
        enable port-mapping on the load balancer ports
  -enable-log-dumping
        store logs to a temporal directory or to the directory specified using the logs-dir flag
  -logs-dir string
        store logs to the specified directory
  -v int
        Verbosity level (default 2)
```
